retrieval/scrape_test.go

```diff
@@ -222,54 +222,42 @@ func TestScrapePoolReload(t *testing.T) {
 	}
 }
 
-func TestScrapePoolReportAppender(t *testing.T) {
+func TestScrapeLoopWrapSampleAppender(t *testing.T) {
 	cfg := &config.ScrapeConfig{
 		MetricRelabelConfigs: []*config.RelabelConfig{
-			{}, {}, {},
+			{
+				Action:       config.RelabelDrop,
+				SourceLabels: model.LabelNames{"__name__"},
+				Regex:        config.MustNewRegexp("a.*"),
+			},
+			{
+				Action:       config.RelabelDrop,
```

Comment:
This is just to have two relabel configs here. (They were empty in the original tests, but the new code actually exercises them, so it would run into a nil pointer panic.)
They don't do anything anyway, as the regexp doesn't match. I can make them different if you prefer, but it would be just for show.